components/TabbedView/TabbedView.unit.test.jsx

```diff
@@ -15,6 +19,24 @@ describe('<TabbedView /> ', () => {
         </TabbedView>,
       );
 
+      expect(toJson(component)).toMatchSnapshot();
+    });
+    it('should match snapshot with badges and icons', () => {
+      const component = mount(
+        <TabbedView>
+          <Tab title="With badge" badge={badge}>
+            With badge content
+          </Tab>
+          <Tab title="With icon" icon={icon}>
+            With icon content
+          </Tab>
+          <Tab title="With icon and badge" icon={icon} badge={badge}>
+            With icon and badge content
+          </Tab>
+          <Tab title="With only text">With only text content</Tab>
+        </TabbedView>,
+      );
+
```

Comment:
What do you think to implement a test with fluid prop?

Reply:
Done